### ISSUE TITLE
java.lang.RuntimeException: Not Found

### ISSUE THREAD

**Issue by hehuanhay** [nemtech/nem2-sdk-java#14]
i used the  demo code to get the accout,like this
`  final AccountHttp accountHttp = new AccountHttp("http://47.106.234.126:7895/account");
        final String address = "SD5DT3-CH4BLA-BL5HIM-EKP2TA-PUKF4N-Y3L5HR-IR54";
        final AccountInfo accountInfo = accountHttp.getAccountInfo(Address.createFromRawAddress(address)).toFuture().get();
        System.out.println(accountInfo);`
there was throwd the exception at the code "toFuture().get()"
`java.lang.RuntimeException: Not Found
	at io.nem.sdk.infrastructure.Http.mapJsonObjectOrError(Http.java:68) ~[sdk-0.9.1.jar:na]
	at io.reactivex.internal.operators.observable.ObservableMap$MapObserver.onNext(ObservableMap.java:57) ~[rxjava-2.2.3.jar:na]
	at io.reactivex.internal.observers.DeferredScalarDisposable.complete(DeferredScalarDisposable.java:82) ~[rxjava-2.2.3.jar:na]
	at io.reactivex.internal.operators.single.SingleToObservable$SingleToObservableObserver.onSuccess(SingleToObservable.java:72) ~[rxjava-2.2.3.jar:na]
	at io.vertx.reactivex.core.impl.AsyncResultSingle.lambda$subscribeActual$0(AsyncResultSingle.java:41) ~[vertx-rx-java2-3.5.0.jar:na]
	at io.vertx.reactivex.ext.web.client.HttpRequest$8.handle(HttpRequest.java:469) ~[vertx-web-client-3.5.0.jar:na]
	at io.vertx.reactivex.ext.web.client.HttpRequest$8.handle(HttpRequest.java:466) ~[vertx-web-client-3.5.0.jar:na]
	at io.vertx.ext.web.client.impl.HttpContext.lambda$null$0(HttpContext.java:125) ~[vertx-web-client-3.5.0.jar:na]
	at io.vertx.core.impl.ContextImpl.lambda$wrapTask$2(ContextImpl.java:344) ~[vertx-core-3.5.0.jar:na]
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute$$$capture(AbstractEventExecutor.java:163) ~[netty-common-4.1.29.Final.jar:4.1.29.Final]
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java) ~[netty-common-4.1.29.Final.jar:4.1.29.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:404) ~[netty-common-4.1.29.Final.jar:4.1.29.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:446) ~[netty-transport-4.1.29.Final.jar:4.1.29.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:884) ~[netty-common-4.1.29.Final.jar:4.1.29.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[netty-common-4.1.29.Final.jar:4.1.29.Final]
	at java.lang.Thread.run(Thread.java:748) [na:1.8.0_172]`
how can i resolve this exception?

**Comment by Suheil123** [comment]
This part of the code is faulty 
final AccountHttp accountHttp = new AccountHttp("http://47.106.234.126:7895/account"). the solution is to remove "/account" from the host address

**Comment by Esquive** [comment]
Hi all I have the same issue with the above AccountInfo code

```
final AccountHttp accountHttp = new AccountHttp("http://23.228.67.85:7890");
                        final String address = "TBQQLDZWVRSCIIQBWUIHCHCWEHBWXECFX745Y73K";
                        final AccountInfo accountInfo = accountHttp.getAccountInfo(Address.createFromRawAddress(address)).toFuture().get();
```

I used the IP address from the node I connect to with the nanowallet (TESTNET). 

I am having the same exception with the Transaction sample code:
[Transaction Sample] (https://nemtech.github.io/guides/transaction/sending-a-transfer-transaction.html)

The Exception is thrown at the get() call:
```
transactionHttp = new TransactionHttp("http://23.228.67.85:7890");
                        Future<TransactionAnnounceResponse> response = transactionHttp.announce(signedTransaction).toFuture();
                        TransactionAnnounceResponse txResponse = response.get();
```

**Comment by fboucquez** [comment]
Hi @Esquive,

We have released a new version, 0.14.0 that works against the latest catapult server. Are you seeing the same problem?

https://github.com/tech-bureau/catapult-service-bootstrap

Cheers
Fernando

**Comment by fboucquez** [comment]
Hi @Esquive 

I'm closing this issue. If you have any other problem please report it. 

Cheers
Fernando
